.travis.yml

```diff
@@ -1,37 +1,50 @@
 language: python
 sudo: false
 python:
-  - 2.7
-  - 3.4
   - 3.5
 cache:
   directories:
     - $HOME/.cache/pip
-env:
-  - DJANGO=django==1.8.14
-  - DJANGO=django==1.8.14 DB_ENGINE="django.db.backends.postgresql_psycopg2" DB_NAME="test_project" DB_USER="postgres"
-  - DJANGO=django==1.8.14 DB_ENGINE="django.db.backends.mysql" DB_NAME="test_project" DB_USER="travis"
-  - DJANGO=django==1.9.9
-  - DJANGO=django==1.9.9 DB_ENGINE="django.db.backends.postgresql" DB_NAME="test_project" DB_USER="postgres"
-  - DJANGO=django==1.9.9 DB_ENGINE="django.db.backends.mysql" DB_NAME="test_project" DB_USER="travis"
-  - DJANGO=django==1.10
-  - DJANGO=django==1.10 DB_ENGINE="django.db.backends.postgresql" DB_NAME="test_project" DB_USER="postgres"
-  - DJANGO=django==1.10 DB_ENGINE="django.db.backends.mysql" DB_NAME="test_project" DB_USER="travis"
 matrix:
   fast_finish: true
 services:
   - postgresql
   - mysql
+env:
+  - TOXENV=py27-django18-sqlite
+  - TOXENV=py27-django18-postgres
+  - TOXENV=py27-django18-mysql
+  - TOXENV=py27-django19-sqlite
+  - TOXENV=py27-django19-postgres
+  - TOXENV=py27-django19-mysql
+  - TOXENV=py27-django110-sqlite
+  - TOXENV=py27-django110-postgres
+  - TOXENV=py27-django110-mysql
+
+  - TOXENV=py34-django18-sqlite
+  - TOXENV=py34-django18-postgres
+  - TOXENV=py34-django18-mysql
+  - TOXENV=py34-django19-sqlite
+  - TOXENV=py34-django19-postgres
+  - TOXENV=py34-django19-mysql
+  - TOXENV=py34-django110-sqlite
+  - TOXENV=py34-django110-postgres
+  - TOXENV=py34-django110-mysql
+
+  - TOXENV=py35-django18-sqlite
+  - TOXENV=py35-django18-postgres
+  - TOXENV=py35-django18-mysql
+  - TOXENV=py35-django19-sqlite
+  - TOXENV=py35-django19-postgres
+  - TOXENV=py35-django19-mysql
+  - TOXENV=py35-django110-sqlite
+  - TOXENV=py35-django110-postgres
+  - TOXENV=py35-django110-mysql
 install:
-  - travis_retry pip install $DJANGO
-  - if [[ "$DB_ENGINE" == "django.db.backends.postgresql" ]] ; then travis_retry pip install psycopg2; fi
-  - if [[ "$DB_ENGINE" == "django.db.backends.postgresql_psycopg2" ]] ; then travis_retry pip install psycopg2; fi
-  - if [[ "$DB_ENGINE" == "django.db.backends.mysql" ]] ; then travis_retry pip install mysqlclient ; fi
-  - travis_retry pip install -e .
+  - pip install tox
 before_script:
-  - if [[ "$DB_ENGINE" == "django.db.backends.mysql" ]] ; then mysql -e 'create database test_project'; fi
-  - if [[ "$DB_ENGINE" == "django.db.backends.postgresql" ]] ; then psql -c 'create database test_project;' -U postgres; fi
-  - if [[ "$DB_ENGINE" == "django.db.backends.postgresql_psycopg2" ]] ; then psql -c 'create database test_project;' -U postgres; fi
-script: python src/tests/runtests.py
+  - mysql -e 'create database test_project'
+  - psql -c 'create database test_project;' -U postgres
+script: tox
```

Comment:
@amureki 

to work correctly in travis-ci, you must explicitly indicate which is the TOXENV you want to run. simple use:
```yml
script: tox -e $TOXENV
```
instead of
```yml
script: tox
```